app/utils/callback/lava.py

```diff
@@ -256,9 +256,9 @@ def _add_boot_log(meta, job_log, base_path, name):
             if e.errno != errno.EEXIST:
                 raise e
 
-    with codecs.open(txt_path, "w", "utf-8") as txt, \
```

Comment:
Alternative using only one `with`:
```
    with (codecs.open(txt_path, "w", "utf-8"),
          codecs.open(html_path, "w", "utf-8")) as txt, html:
        utils.lava_log_parser.run(log, meta, txt, html)
```
but using 2 nested `with` statements is fine too.